homeassistant/components/octoprint/__init__.py

```diff
@@ -51,10 +55,28 @@ def ensure_valid_path(value):
     return value
 
 
+@dataclass
+class OctoprintBinarySensorMetadata(BinarySensorEntityDescription):
+    """Metadata for an individual Skybell binary_sensor."""
+
+    endpoint: str = ""
+    group: str = ""
+
+
 BINARY_SENSOR_TYPES = {
```

Comment:
This can be come a tuple

Reply:
No not unless I add conversion code, where it is used because "Printing Error" is not a legal variable name.